source/manual/fix-out-of-date-search-indices.html.md

```diff
@@ -15,21 +15,22 @@ and `unpublish` messages that have not been processed need to be resent.
 Content in the `govuk` index is populated from the [Publishing API message queue][queue].
 Missing documents can be recovered by resending the content to the message queue. In the
 Publishing API, run the following rake task (including the quotes) to replay traffic between
-two datestamps:
+two datestamps (after SSH'ing into the `publishing_api` machine):
 
 ```
+cd /var/apps/publishing-api && \
 govuk_setenv publishing-api bundle exec \
 rake 'represent_downstream:published_between[2018-12-17T01:02:30, 2018-12-18T10:20:30]'
```

Comment:
Should we now remove the command line option from the docs, as the standard / preferred approach is to run Rake tasks from Jenkins?

Reply:
Agreed. I've tacked on a couple of commits to tidy this up.